doc/transfer-format.rst

```diff
@@ -107,4 +107,4 @@ Setup Code ``D9VN-RD7A-7T3B-BWG7-LEPY-3NYT``.
   xUbOVydoyTLaP9bQScqv4WBqVcw5k8v4i+I=
   =k6Rc
   -----END PGP MESSAGE-----
-  </pre><hr><i><a href="http://example.org/">Generated by Autocrypt-capable Client</a>.</i></body></html>
+  </pre><hr><i><a href="http://example.org/">Generated by Autocrypt-capable MUA</a>.</i></body></html>
```

Comment:
This looks like user-facing stuff -- we don't want to include "MUA" here, do we?  This is a fake ad/promotion for the "example" MUA.  We should make the ad look more plausible than this.